tests/app/main/test_briefs.py

```diff
@@ -1900,110 +1958,289 @@ def test_view_response_result_page_escapes_essential_and_nice_to_have_requiremen
         assert "**n2h two with markdown**" in data
         assert "<strong>n2h two with markdown</strong>" not in data
 
-    def test_view_response_result_submitted_ok(self, data_api_client):
+    def test_view_response_result_submitted_ok_if_live_or_closed(self, data_api_client):
+        self.set_framework_and_eligibility_for_api_client(data_api_client)
+        data_api_client.find_brief_responses.return_value = self.brief_responses
+        brief_copy = self.brief.copy()
+
+        for status in ('live', 'closed'):
+            brief_copy['briefs']['status'] = status
+            data_api_client.get_brief.return_value = brief_copy
+
+            res = self.client.get('/suppliers/opportunities/1234/responses/result')
+
+            assert res.status_code == 200
+            doc = html.fromstring(res.get_data(as_text=True))
+            assert doc.xpath('//p[contains(@class, "banner-message")]')[0].text.strip() == \
+                "Your application has been submitted."
+
+    def test_view_response_result_submitted_unsuccessful(self, data_api_client):
+        self.brief_responses['briefResponses'][0]['essentialRequirements'][1] = False
+        self.set_framework_and_eligibility_for_api_client(data_api_client)
         data_api_client.get_brief.return_value = self.brief
-        data_api_client.get_framework.return_value = self.framework
-        data_api_client.is_supplier_eligible_for_brief.return_value = True
-        data_api_client.find_brief_responses.return_value = {
-            "briefResponses": [
-                {"essentialRequirements": [True, True, True]}
-            ]
-        }
+        data_api_client.find_brief_responses.return_value = self.brief_responses
         res = self.client.get('/suppliers/opportunities/1234/responses/result')
 
         assert res.status_code == 200
         doc = html.fromstring(res.get_data(as_text=True))
-        assert doc.xpath('//h1')[0].text.strip() == \
-            "Your response to ‘I need a thing to do a thing’ has been sent"
+        assert doc.xpath('//h1')[0].text.strip() == "Your application for ‘I need a thing to do a thing’"
```

Comment:
Is there nothing else that needs to be asserted in this case? Such as no 'Your application has been submitted' banner message or an alternative banner message?